nodejs-jquery/nodejs-jquery.spec

```diff
@@ -0,0 +1,57 @@
+%{?scl:%scl_package nodejs-%{npm_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global npm_name jquery
+
+Name: %{?scl_prefix}nodejs-%{npm_name}
+Version: 1.11.3
+Release: 1%{?dist}
+Summary: JavaScript library for DOM operations
+License: MIT
+Group: Development/Libraries
+URL: https://github.com/jquery/jquery.git
+Source0: http://registry.npmjs.org/jquery/-/jquery-1.11.3.tgz
+BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}-root-%(%{__id_u} -n)
+Requires: nodejs(engine)
```

Comment:
> The name uses SCL macros so it's served as part of our SCL (tfm-nodejs-jquery), other packages like tfm-nodejs-jquery-ujs can have a Requires: %{?scl_prefix}npm(jquery).

The files are not packaged in the SCL root though, so there's no point in just having the prefix. When it's using the non-SCL NodeJS build/devel packages and paths, the SCL prefix should be removed to match.

> nodejs(engine), nodejs-devel and nodejs-packaging are served already in all of our distributions, I don't think we should put these in the SCL, we can require them wherever.

Yeah, that's fine, it doesn't make much difference using the SCL or non-SCL version as they both appear to be maintained.